queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animal WHERE name LIKE '%mon';
+SELECT name from animal WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name from animal WHERE neutered = 't' AND escape_attempts < 3;
+SELECT date_of_birth from animal WHERE name IN ('Agumon', 'Pikachu');
+SELECT name, escape_attempts from animal WHERE weight_kg > 10.5;
+SELECT * from animal WHERE neutered = 't';
+SELECT * from animal WHERE name <> 'Gabumon';
+SELECT * from animal WHERE weight_kg BETWEEN 10.4 AND 17.3;
```

Comment:
- [ ] Great job performing transactions! I would suggest doing all the transactions required by the activity, there are many requirements missing:
- All the `DELETE` transactions.
- All the `BEGIN`, `ROLLBACK` and `COMMIT` functions.
- All the `AVG()` and `COUNT()` functions.
- All the `GROUP BY` functions.
- All the `MIN()` and `MAX()` functions.

Reply:
Ok @smunozmo Thank you, I will do them now!